### ISSUE TITLE
Ubuntu 15.04 and systemd

### ISSUE THREAD

**Issue by nicl** [guardian/machine-images#16]
Wanted to gauge views on systemd. I'm not sure there are significant benefits against upstart for our typical use cases of one-service a VM, and logs shipping off instance, but it is _the future_.

**Comment by sihil** [comment]
Yes. We should. Given the size of the change, I suggest we build another set of images in tandem.

**Comment by nicl** [comment]
@sihil agreed, I'll see if Discussion are happy to go for this for our new stuff. If so, I'll add a PR soon!

**Comment by sihil** [comment]
@philwills has been converting the status-app to systemd already - so that's likely a good place to start.

**Comment by philwills** [comment]
Yea, I'd be happy to pair with someone on this, as I've done it in a non-packer kind of way, for status-app, but would like to get it in here.

**Comment by rich-nguyen** [comment]
We've only recently migrated to upstart. Bring on systemd! What are the benefits we care about?

**Comment by philwills** [comment]
Compare 

https://github.com/guardian/status-app/blob/master/status-app.conf

to

https://github.com/guardian/status-app/blob/master/status-app.service

The latter seems much more descriptive of what's happening to me, as well as being more terse.

There's some nice integration of logging, so for instance if you look at the status of your app, you get a nice process tree view and tail of log like

![image](https://cloud.githubusercontent.com/assets/68329/7701299/bf7ab89c-fe1d-11e4-9fcd-890241170624.png)

Also, upstart isn't available on 15.04 and later.

**Comment by philwills** [comment]
@tackley is very enthused by systemd, so might want to add something.

I've just released a version of https://github.com/guardian/riffraff-artifact that will default to copying systemd service files into your RiffRaff artifact.

**Comment by cb372** [comment]
Just a heads-up that I've started work today on adding a Vivid image.

**Comment by philwills** [comment]
Whilst we putting in some heads up, I've got som very exciting :exclamation: auto-generation of systemd service files as part of a deb build going on in status-app:

https://github.com/guardian/status-app/blob/native-packager-deb/build.sbt
